### ISSUE TITLE
Tooltip - Needs role='tooltip'

### ISSUE THREAD

**Issue by Christopher-William-Guan** [IBM/carbon-components-react#1154]
## Detailed description

> Describe in detail the issue you're having.
I am trying to making our product meet accessibility guidelines and running the IBM Accessibility Research Dynamic Assessment Plugin. The scan reveals that an 'aria-expanded' is not a valid attribute for a div tag when it does not have a role='tooltip'. Adding this attribute would get rid of the violation (see screenshots below)

> Is this a feature request (new component, new icon), a bug, or a general issue?
Bug

> Is this issue related to a specific component?
Tooltip

> What browser are you working in?
Google Chrome

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?
APM

## Additional information
We have 50 items in a table that utilize the tooltip. As you can see if the screenshot, after directly adding the role='tooltip' to the div tag, the number of violations go down. 

<img width="1160" alt="screen shot 2018-08-02 at 9 16 24 am" src="https://user-images.githubusercontent.com/42038154/43589910-562fafa6-9635-11e8-893a-a74d81ffebc9.png">
<img width="1161" alt="screen shot 2018-08-02 at 9 17 11 am" src="https://user-images.githubusercontent.com/42038154/43589913-5aaf9a46-9635-11e8-9ff2-b2531e6ad305.png">

**Comment by asudoh** [comment]
Hi @Christopher-William-Guan thank you for reporting! If you have any quick fix, feel free to send a PR our way.
